docs/installation-guide.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
@Florence-Njeri Hey, so a reminder again that we use sentence capitalization in headers for Docs. (I keep having to update your headers girl 😜😜😜😜)

Reply:
I'm sorry, I promise it won't happen again 😅